Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1289,7 +1334,9 @@ public extension Purchases {
                   storeKit2Setting: storeKit2Setting,
                   storeKitTimeout: storeKitTimeout,
                   networkTimeout: networkTimeout,
-                  dangerousSettings: dangerousSettings)
+                  dangerousSettings: dangerousSettings,
+                  showStoreMessagesAutomatically: showStoreMessagesAutomatically
+                 )
```

Comment:
Super-nit: since the opening `(` is in the same line as the first parameter, this should match and be on the same line as the last.